### ISSUE TITLE
Unable to install version 23

### ISSUE THREAD

**Issue by pru-brennaveen** [pouyakary/comment#28]
I am getting this error when trying to update to version 23.  I am currently on version 22.

`2023-03-27 08:09:54.046 [error] ENOENT: no such file or directory, open 'c:\Users\MyUserName\.vscode\extensions\.43a79c12-55b8-4e1d-bd07-514d6631eeca\Icon
.original': Extract: ENOENT: no such file or directory, open 'c:\Users\MyUserName\.vscode\extensions\.43a79c12-55b8-4e1d-bd07-514d6631eeca\Icon
.original'
    at x.u (vscode-file://vscode-app/c:/Users/MyUserName/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:15438)
    at async x.extractUserExtension (vscode-file://vscode-app/c:/Users/MyUserName/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:12665)
    at async X.u (vscode-file://vscode-app/c:/Users/MyUserName/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:19248)
    at async X.x (vscode-file://vscode-app/c:/Users/MyUserName/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:20836)
    at async X.h (vscode-file://vscode-app/c:/Users/MyUserName/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:18897)`

**Comment by pouyakary** [comment]
That's strange! Comment doesn't do anything strange actually. It should be some sort of conflict in your VS Code. can you erase your app data and remove the extension folder and try again?

**Comment by pru-brennaveen** [comment]
Unfortunately, that did not help.  I am still seeing the same error.  I even uninstalled vscode and removed all of the remaining data references I could find in AppData, plus deleted everything in the .vscode folder on my profile, but that did not fix it either.  I am seeing the issue on two separate Windows 11 computers.

**Comment by pouyakary** [comment]
Wow that is really strange. I'll get back to you in about a week to see if I can fix it

**Comment by elazar55** [comment]
I'm getting the same error.

**Comment by biiigwang** [comment]
I'm facing the same issue.

**Comment by stelladraco27** [comment]
Same issue as well. Also reinstalled VSCode.

Here is the log:
`2023-05-07 15:02:19.550 [warning] Via 'product.json#extensionEnabledApiProposals' extension 'ms-vscode.vscode-selfhost-test-provider' wants API proposal 'testContinuousRun' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2023-05-07 15:02:20.572 [warning] [twxs.cmake]: Cannot register 'cmake.cmakePath'. This property is already registered.
2023-05-07 15:02:21.861 [error] [Extension Host] (node:24436) [PINODEP007] PinoWarning: bindings.level is deprecated, use options.level option instead
(Use `Code --trace-warnings ...` to show where the warning was created)
2023-05-07 15:02:22.142 [error] [Extension Host] (node:24436) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
2023-05-07 15:02:22.605 [info] [perf] Render performance baseline is 43ms
2023-05-07 15:02:23.039 [error] SyntaxError: Unexpected token 'A', "AQAAANCMnd"... is not valid JSON
    at JSON.parse (<anonymous>)
    at e.MainThreadSecretState.$getPassword (vscode-file://vscode-app/i:/InstalledApps/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:1525:48130)
2023-05-07 15:02:24.232 [error] ENOENT: no such file or directory, open 'c:\Users\stell\AppData\Roaming\Code\User\globalStorage\dendron.dendron\segmentresidualcache.log': Error: ENOENT: no such file or directory, open 'c:\Users\stell\AppData\Roaming\Code\User\globalStorage\dendron.dendron\segmentresidualcache.log'
2023-05-07 15:02:44.250 [error] Extract: ENOENT: no such file or directory, open 'c:\Users\stell\.vscode\extensions\.5ecc7035-6420-4c3c-8491-50459a66736d\Icon
.original'
    at e.ExtensionsScanner.u (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:15416)
    at async e.ExtensionsScanner.extractUserExtension (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:12651)
    at async x.v (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:19429)
    at async x.y (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:21039)
    at async x.h (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:19004)
2023-05-07 15:02:44.277 [error] ENOENT: no such file or directory, open 'c:\Users\stell\.vscode\extensions\.5ecc7035-6420-4c3c-8491-50459a66736d\Icon
.original': Extract: ENOENT: no such file or directory, open 'c:\Users\stell\.vscode\extensions\.5ecc7035-6420-4c3c-8491-50459a66736d\Icon
.original'
    at e.ExtensionsScanner.u (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:15416)
    at async e.ExtensionsScanner.extractUserExtension (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:12651)
    at async x.v (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:19429)
    at async x.y (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:21039)
    at async x.h (i:\InstalledApps\Microsoft VS Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:88:19004)`

**Comment by stelladraco27** [comment]
This issue arises when I try to install your extension. Based on the log, it seems that the installer for your extension may be bugged/missing a file (that icon) when it tried to extract files and attempt to install itself.

**Comment by stelladraco27** [comment]
Might just be a bad build.

**Comment by LucasFA** [comment]
Related to this, I can't even clone the repository correctly. I get this:
<details>

```bash
git clone https://github.com/LucasFA/comment.git
Cloning into 'comment'...
remote: Enumerating objects: 1259, done.
remote: Counting objects: 100% (468/468), done.
remote: Compressing objects: 100% (260/260), done.
Receiving objects:  97% (1222/1259), 564.01 KiB | 754.00 KiB/seused 791Receiving objects:  95% (1197/1259), 564.01 KiB | 754.00 KiB/s
Receiving objects: 100% (1259/1259), 1006.28 KiB | 1.05 MiB/s, done.
Resolving deltas: 100% (659/659), done.
error: invalid path 'Icon?'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

</details>

and if I
<details>
<summary> git status </summary>

```bash
On branch master
Your branch is up to date with 'origin/master'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        deleted:    .DS_Store
        deleted:    .eslintrc.json
        deleted:    .gitignore
        deleted:    .vscode/extensions.json
        deleted:    .vscode/launch.json
        deleted:    .vscode/settings.json
        deleted:    .vscode/tasks.json
        deleted:    .vscodeignore
        deleted:    .yarnrc
        deleted:    CHANGELOG.md
        deleted:    "Icon\r"
        deleted:    "Icon\r.original"
        deleted:    LICENSE.txt
        deleted:    README.md
        deleted:    icon.png
        deleted:    package.json
        deleted:    src/web/engine/concepts/context.ts
        deleted:    src/web/engine/concepts/indentation.ts
        deleted:    src/web/engine/concepts/index.ts
        deleted:    src/web/engine/concepts/language.ts
        deleted:    src/web/engine/concepts/ornament.ts
        deleted:    src/web/engine/constants.ts
        deleted:    src/web/engine/extraction/content.ts
        deleted:    src/web/engine/extraction/index.ts
        deleted:    src/web/engine/generators/index.ts
        deleted:    src/web/engine/generators/separator.ts
        deleted:    src/web/engine/generators/skeleton.ts
        deleted:    src/web/engine/generators/title.ts
        deleted:    src/web/engine/generators/types.ts
        deleted:    src/web/engine/index.ts
        deleted:    src/web/engine/protocols/configurations.ts
        deleted:    src/web/engine/protocols/environment.ts
        deleted:    src/web/engine/protocols/index.ts
        deleted:    src/web/engine/protocols/language.ts
        deleted:    src/web/engine/toolkit/capitalize.ts
        deleted:    src/web/engine/toolkit/index.ts
        deleted:    src/web/engine/toolkit/separator.ts
        deleted:    src/web/engine/validation/index.ts
        deleted:    src/web/engine/validation/name.ts
        deleted:    src/web/engine/validation/size.ts
        deleted:    src/web/engine/validation/types.ts
        deleted:    src/web/extension.ts
        deleted:    src/web/test/suite/index.ts
        deleted:    src/web/vscode/contributions/generate.ts
        deleted:    src/web/vscode/contributions/index.ts
        deleted:    src/web/vscode/contributions/rename.ts
        deleted:    src/web/vscode/languages/dictionary.ts
        deleted:    src/web/vscode/languages/index.ts
        deleted:    src/web/vscode/parameters/editor.ts
        deleted:    src/web/vscode/parameters/index.ts
        deleted:    src/web/vscode/parameters/settings.ts
        deleted:    src/web/vscode/services/generators.ts
        deleted:    src/web/vscode/services/index.ts
        deleted:    src/web/vscode/tools/error.ts
        deleted:    src/web/vscode/tools/index.ts
        deleted:    src/web/vscode/tools/replace.ts
        deleted:    tsconfig.json
        deleted:    vsc-extension-quickstart.md
        deleted:    webpack.config.js
        deleted:    yarn.lock
```

</details>

**Comment by LucasFA** [comment]
@pouyakary I don't think you have `vsce publish patch`'d this yet

**Comment by ccll** [comment]
@pouyakary 
I'm afraid to say the issue still persists.
Would you mind publishing the fix to the marketplace?
Thanks so much.

**Comment by pouyakary** [comment]
I just released a new version `23.1.0`. Hope that you see the result there.
